src/synology_dsm/api/download_station/__init__.py

```diff
@@ -32,17 +37,50 @@ def update(self):
                 self._tasks_by_id[task_data["id"]] = SynoDownloadTask(task_data)
 
     # Global
+    def update_info(self):
+        """Update info about the Download Station instance."""
+        self._info = self._dsm.get(self.INFO_API_KEY, "GetInfo")["data"]
+
     def get_info(self):
         """Return general informations about the Download Station instance."""
-        return self._dsm.get(self.INFO_API_KEY, "GetInfo")
+        return self._info
+
+    def update_config(self):
+        """Update configuration about the Download Station instance."""
+        self._config = self._dsm.get(self.INFO_API_KEY, "GetConfig")["data"]
 
     def get_config(self):
         """Return configuration about the Download Station instance."""
-        return self._dsm.get(self.INFO_API_KEY, "GetConfig")
+        return self._config
+
+    def update_schedule_config(self):
+        """Update schedule configuration about the Download Station instance."""
+        self._schedule_config = self._dsm.get(self.SCHEDULE_API_KEY, "GetConfig")[
+            "data"
+        ]
+
+    def get_schedule_config(self):
+        """Return schedule configuration about the Download Station instance."""
+        return self._schedule_config
+
+    def set_schedule_config(self, enabled: bool = None, emule_enabled: bool = None):
+        """Set schedule configuration about the Download Station instance."""
+        config = {}
+        if enabled is not None:
+            config["enabled"] = enabled
+        if emule_enabled is not None:
+            config["emule_enabled"] = emule_enabled
+        res = self._dsm.get(self.SCHEDULE_API_KEY, "SetConfig", config)
+        self.update_schedule_config()
+        return res
+
+    def update_stat(self):
+        """Update statistic about the Download Station instance."""
+        self._stat = self._dsm.get(self.STAT_API_KEY, "GetInfo")["data"]
```

Comment:
- this could cause KeyError, in case _dsm.get() was unsuccessful
- prior result was not restricted to content of `data`